decidim-proposals/lib/decidim/proposals/component.rb

```diff
@@ -151,7 +149,12 @@
         answered_at: Time.current,
         published_at: Time.current
       )
-      proposal.add_coauthor(author, user_group: user_group) if n.even?
+      if n.positive?
```

Comment:
Won't `n` be always positive? Why not keep the previous `n.even`?

Reply:
First `n` will be zero: for officialized proposals. Then each proposal will have `n` coauthors.

This way we will have proposals with "all" coauthorships: official, 1 author, many coauthors.

Reply:
Oh, I see! Nice!